### ISSUE TITLE
Translating Actispy

### ISSUE THREAD

**Issue by airon90** [mihi-tr/actispy#15]
Is it possible to make Actispy translatable?

**Comment by mihi-tr** [comment]
I've recently worked on translating with jed - so the answer is yes. Would you be willing to translate?

**Comment by airon90** [comment]
Yes, I want to translate it into Italian, my mothertongue languae, and Esperanto.
I suggest you to read [this page](https://developer.mozilla.org/en-US/Apps/Build/Localization/App_Localization_with_Transifex) about localization. :)

**Comment by mihi-tr** [comment]
Thanks for the pointer - I will prepare the app for translations according do the guide and will let you know as soon as it is ready. Looking forward to Italian and Esperanto versions of the app!

**Comment by airon90** [comment]
Thank you very much!
